pkgs/top-level/all-packages.nix

```diff
@@ -1038,6 +1038,8 @@ with pkgs;
     stdenv = gcc10StdenvCompat;
   };
 
+  adl = callPackage ../applications/video/adl { };
```

Comment:
You should have added `adl` to the applications (search `### APPLICATIONS`) section not tools. (And remember to keep alphabetical order.)

Reply:
Sorry, I didn't notice I did that...